src/Operators/finitedifference.jl

```diff
@@ -2106,13 +2106,32 @@ function Base.Broadcast.materialize!(
     )
 end
 
+
+unwrap_ref(r::Ref) = r[]
+unwrap_ref(r) = r
```

Comment:
This is to work around the fact that `column(::SpectralElementField, inds...)` returns a `Ref`, but `column(::Real, inds...)` gives a `Real`. We should make these consistent.

Reply:
we should  make Tensor / Vector ops act like scalars in broadcasting IMO